tools/packaging/static-build/scripts/qemu-build-post.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script process QEMU post-build.
+#
+set -e
+
+script_dir="$(realpath $(dirname $0))"
+source "${script_dir}/../qemu.blacklist"
+
+if [[ -z "${QEMU_TARBALL}" || -z "${QEMU_DESTDIR}" ]]; then
+	echo "$0: needs QEMU_TARBALL and QEMU_DESTDIR exported"
+	exit 1
+fi
+
+cd "${QEMU_DESTDIR}"
```

Comment:
could you please use `pushd` and `popd`?

Reply:
@devimc sure. I will send another version with this change.

Reply:
@devimc btw, commands inside pushd/popd should be indented? Wondering if there exist a coding style for that.

Reply:
> @devimc btw, commands inside pushd/popd should be indented? Wondering if there exist a coding style for that.

afaik, no